ipmi/Sensor Discovery/doc/README.head.md

```diff
@@ -18,9 +18,19 @@ required by most of the [IPMI templates](../../).
    script into your Zabbix servers/proxy external script directory
    ([`ExternalScripts`](https://www.zabbix.com/documentation/3.0/manual/appendix/config/zabbix_server)).
 3. Make sure the script is executable by the user running the zabbix-server
-4. If you have SELinux active, make sure that the Zabbix-Server is allowed to
-   execute the scripts (you might need to load the provided
-   [SELinux policy](../selinux/rabezbxsensordiscovery.te)). 
+4. If you have SELinux active, make sure that the Zabbix server is allowed to
+   execute the script and is allowed to operate on directories and files within
+   its home directory (this is required for `ipmi-sensor`'s cache
+   (`~/.freeipmi/).
+   On CentOS 7 the following steps were necessary:
+   ```bash
+   # Create the missing home directory of the zabbix user
+   mkdir -m 750 /var/lib/zabbix
+   chown zabbix:zabbix /var/lib/zabbix
+
+   # Set the existing SELinux user and type 
+   chcon -u system_u -t zabbix_var_lib_t /var/lib/zabbix/
```

Comment:
> Running restorecon -Rv /var/lib/zabbix after mkdir/chown should be enough if everything is as I expect.

Indeed, thanks! I've tested the it on a Zabbix server and updated the README accordingly.